src/components/view/sources/sources.ts

```diff
@@ -0,0 +1,50 @@
+import { Source } from '../../models/index.model';
+import './sources.css';
+
+class Sources {
+  private filter: string = 'A';
+  draw(data: Array<Source>) {
+    const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+    const fragment = document.createDocumentFragment();
+    const sourceItemTemp: HTMLTemplateElement | null = document.querySelector('#sourceItemTemp');
+
+    const drawData = () => {
```

Comment:
`drawData ` will be created each time when `draw` method is called, may be it's worth to carry it out?